shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/network/target/RefreshTokenType.kt

```diff
@@ -0,0 +1,28 @@
+package co.nimblehq.blisskmmic.data.network.target
+
+import co.nimblehq.blisskmmic.BuildKonfig
+import co.nimblehq.blisskmmic.data.network.helpers.TargetType
+import io.ktor.http.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+class RefreshTokenType(refreshToken: String):
+    TargetType<RefreshTokenType.RefreshTokenInput> {
+
+    @Serializable
+    data class RefreshTokenInput(
+        @SerialName("grant_type") val grantType: String,
```

Comment:
@blyscuit It would be better if we add more line breaks for easier readability, what do you think?
```suggestion
        @SerialName("grant_type") 
        val grantType: String,
```

Reply:
This was a conversation with @minhnimble long time ago, I don't remember the PR. We concluded that we will use this format in this project.

Reply:
I like this one most, but I had no valid argument for it, just preference. ❤️ 
```
class PaginationMetaApiModel(
    val page: Int,
    val pages: Int,
    @SerialName("page_size")
    val pageSize: Int,
    val records: Int
)
```

Reply:
@suho @blyscuit @doannimble actually from our [Compass](https://nimblehq.co/compass/development/code-conventions/kotlin/#blank-line), there is a guideline from Ribot that mention about this [annotation case for fields](https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md#226-annotations) as follow:
`Annotations applying to fields should be listed on the same line, unless the line reaches the maximum line length.`

-> Hope that clear you guys confusion if any 🙏 And maybe we can create a PR to bring this convention to our Compass for transparency 😁